experiment/simulation/js/main.js

```diff
@@ -1 +1,460 @@
-//Your JavaScript goes in here
+//instructions for the experiment
+let messages=[
+"Hover over the 4-bit table and Select any two rows to add",
+"Results are highlighted green and overflow is displayed in result table\nClick on 5-bit table to switch",
+"Click on reset to clear and restart anytime",
+];
+let mat = [];
+let temp = [];
+let tobeadded = [];
+let selectedrow = [];
+let coloredidx = [];
+let resultarray = [0,0,0,0,0];
+let visitedarray = [0,0,0,0,0]; 
+let resultarrayname = ['rowbinar','row2s','row1s','rowunsigned','rowsigned'];
+vis = [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0];
+sameelemflag = 0;  //used to fix the first selected row and change the second selected row if want
+let glob = 0;
+let flag = 0;
+let afteradd = 0;
+let message = messages[0];
+document.getElementById("message").innerHTML = message;
+//printng overflow in the result table
+function addoverflow()
+{
+    for (var i=0;i<5;i++)
+    {
+        console.log(resultarray[i]);
+    }
+    console.log("inside addrow id=")
```

Comment:
Please remove all console.log statement from the final code. If necessary, use console.debug in place of console.log